cwms-data-api/src/test/java/cwms/cda/data/dto/location/kind/GateChangeTest.java

```diff
@@ -199,6 +199,7 @@ static GateChange buildTestGateChange(CwmsId projectId, Instant changeDate, Gate
                                        .withNotes(notes)
                                        .withChangeDate(changeDate)
                                        .withSettings(settings)
+//                                       .withType("gate-change")
```

Comment:
Either delete if not needed, or provide explanation of why it's sitting here commented out (in the comment itself.)

Reply:
Deleted unnecessary comment